### PR TITLE
Fixed build and tests

### LINKED ISSUE #566: Error running build.ps1: Invalid restore input. UAP projects must contain exactly one target framework.
I'm trying to run the `build.ps1` script and it is failing. Full output below. Fails on both the `master` and `devel` branches with the same error. Any advice?

```
C:\c\AngleSharp [devel]> .\build.ps1
Feeds used:
  C:\Users\jpoehls\.nuget\packages\
  https://api.nuget.org/v3/index.json
  C:\iwcache\nuget
  C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\

All packages listed in C:\c\AngleSharp\tools\packages.config are already installed.
Analyzing build script...
Processing build script...
Installing addins...
Compiling build script...

----------------------------------------
Setup
----------------------------------------
Executing custom setup action...
Building version 0.10.0 of AngleSharp.
For the publish target the following environment variables need to be set:
  NUGET_API_KEY, GITHUB_API_TOKEN

========================================
Clean
========================================
Executing task: Clean
Cleaning directory C:/c/AngleSharp/src/AngleSharp/bin
Cleaning directory C:/c/AngleSharp/bin/0.10.0
Creating directory C:/c/AngleSharp/bin/0.10.0/nuget
Finished executing task: Clean

========================================
Restore-Packages
========================================
Executing task: Restore-Packages
MSBuild auto-detection: using msbuild version '15.1.1012.6693' from 'C:\Program Files (x86)\Microsoft Visual Studio\2017\Professional\MSBuild\15.0\bin'.
All packages listed in packages.config are already installed.
Invalid restore input. UAP projects must contain exactly one target framework. Input files: C:\c\AngleSharp\src\AngleSharp\AngleSharp.Core.csproj, C:\c\AngleSharp\src\AngleSharp\project.json.
An error occured when executing task 'Restore-Packages'.
Error: NuGet: Process returned an error (exit code 1).
```

### PR DESCRIPTION
**Pull request by laurynasr**
Resolve #566
Use newer version of Cake, as Cake.FileHelpers 2.0.0 is incompatible with Cake 0.14.0
Added global.json to use a specific version of .NET Core SDK
Use nuget 3.5.0 as newer versions error out when trying to restore a project that contains both .csproj and project.json files
Fixed test failures

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "1.0.0-preview2-1-003177"
```
Comment (FlorianRappl) [inline global.json:3]: Woot! This is it! Thanks a lot man :) 👍

**Comment by FlorianRappl** [comment]
Really good work. Thanks a lot! Now the only thing we need to figure out is if we push it directly to master or to the v0.9.9.1 (hotfix) branch. 

I think to spare you the trouble of re-PRing I will just merge it into `master` right now, sync with hotfix and prepare a new version.

Much appreciated! 🍻

**Comment by laurynasr** [comment]
Sorry for messing up the process, I'll try to do better next time ;)

**Comment by FlorianRappl** [comment]
No its fine, `devel` is blocked with v0.10, so putting it into `master` was great.

**Pull request merged**